sdk/src/logs/logger_provider_factory.cc

```diff
@@ -0,0 +1,58 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#ifdef ENABLE_LOGS_PREVIEW
+
+#  include "opentelemetry/sdk/logs/logger_provider_factory.h"
+#  include "opentelemetry/sdk/logs/logger_provider.h"
+#  include "opentelemetry/sdk/resource/resource.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace sdk
+{
+namespace logs
+{
+
+std::unique_ptr<opentelemetry::logs::LoggerProvider> LoggerProviderFactory::Create(
+    std::unique_ptr<LogProcessor> &&processor)
+{
+  auto resource = opentelemetry::sdk::resource::Resource::Create({});
+  return Create(std::move(processor), resource);
```

Comment:
I think move resource here will have a better performance?

Reply:
The resource parameter in Create is a `const &`, there is no copy.